doc/nrf/releases/release-notes-latest.rst

```diff
@@ -400,6 +402,8 @@ The following list summarizes the most important changes inherited from upstream
   * Removed the legacy DT macros.
   * Started exposing dependency ordinals for walking the dependency hierarchy.
   * Added documentation for the :ref:`DTS bindings <zephyr:devicetree_binding_index>`.
+  * Added the ``UICR`` and ``FICR`` peripherals to the Devicetree.
+  * Interrupt priorities in Devicetree on Nordic devices now default to ``NRF_DEFAULT_IRQ_PRIORITY`` instead of hard-coded values.
```

Comment:
```suggestion
  * Changed the interrupt priorities in devicetree for Nordic Semiconductor devices to default to ``NRF_DEFAULT_IRQ_PRIORITY`` instead of hard-coded values.
```